tests/test_more.py

```diff
@@ -145,3 +145,54 @@ def test_dedup_handler(logger):
     assert 2 == len(test_handler.records)
     assert 'message repeated 2 times: foo' in test_handler.records[0].message
     assert 'message repeated 1 times: bar' in test_handler.records[1].message
+
+
+class TestRiemannHandler(object):
+
+    @require_module("riemann_client")
+    def test_happy_path(self, logger):
```

Comment:
The tests you've added don't run on Travis, because they require a library that's missing there (Given you haven't made any changes to the .travis.yml file). This means that if someone breaks the handler at some point, we won't get notified because no tests run on it...